src/main/scala/exercises/06_Collections.scala

```diff
@@ -139,23 +145,31 @@ object Collections {
    * As usual, various ways exist: pattern matching, folding, ...
    */
   def maxElementInList(l: List[Int]): Int = {
-    error("fix me")
+    l.foldLeft(l.head)((acc: Int, max: Int) => if (acc > max) acc else max)
   }
 
   /**
    * Calculate the sum of the equally position elements
    * of the two list
    */
   def sumOfTwo(l1: List[Int], l2: List[Int]): List[Int] = {
-    error("fix me")
+    if (l1.isEmpty)
+      l2
+    else if (l2.isEmpty)
+      l1
+    else
+      (l1.head+l2.head)+:sumOfTwo(l1.tail,l2.tail)
   }
 
   /**
    *  For this exercise preferably make use of the sumOfTwo
    * method above
    */
   def sumOfMany(l: List[Int]*): List[Int] = {
-    error("fix me")
+    if (l.isEmpty)
```

Comment:
Same comment as above. This could also be written with a higher-order fold function!